bellows/ezsp.py

```diff
@@ -31,17 +32,31 @@ def connect(self, device):
     def reset(self):
         return self._gw.reset()
 
+    @asyncio.coroutine
+    def version(self, version):
+        result = yield from self._command('version', version)
+        if result[0] != version:
+            LOGGER.debug("Switching to version %d", result[0])
+            result = self._command('version', result[0])
```

Comment:
Is this to instruct the NCP that you're switching versions?

Any reason you're returning the future from _command here, and not yielding inside this coroutine?